sdk/python/feast/repo_config.py

```diff
@@ -51,23 +51,36 @@
+    "feast.infra.online_stores.contrib.ikv_online_store.ikv.IKVOnlineStore": "feast.infra.online_stores.ikv_online_store.ikv.IKVOnlineStore",
+    "feast.infra.online_stores.contrib.elasticsearch.ElasticSearchOnlineStore": "feast.infra.online_stores.elasticsearch_online_store.ElasticSearchOnlineStore",
+    "feast.infra.online_stores.contrib.singlestore_online_store.singlestore.SingleStoreOnlineStore": "feast.infra.online_stores.singlestore_online_store.singlestore.SingleStoreOnlineStore",
+    "feast.infra.online_stores.contrib.qdrant.QdrantOnlineStore": "feast.infra.online_stores.cqdrant.QdrantOnlineStore",
+}
+
 ONLINE_STORE_CLASS_FOR_TYPE = {
     "sqlite": "feast.infra.online_stores.sqlite.SqliteOnlineStore",
     "datastore": "feast.infra.online_stores.datastore.DatastoreOnlineStore",
     "redis": "feast.infra.online_stores.redis.RedisOnlineStore",
     "dynamodb": "feast.infra.online_stores.dynamodb.DynamoDBOnlineStore",
     "snowflake.online": "feast.infra.online_stores.snowflake.SnowflakeOnlineStore",
     "bigtable": "feast.infra.online_stores.bigtable.BigtableOnlineStore",
-    "postgres": "feast.infra.online_stores.contrib.postgres.PostgreSQLOnlineStore",
-    "hbase": "feast.infra.online_stores.contrib.hbase_online_store.hbase.HbaseOnlineStore",
-    "cassandra": "feast.infra.online_stores.contrib.cassandra_online_store.cassandra_online_store.CassandraOnlineStore",
-    "mysql": "feast.infra.online_stores.contrib.mysql_online_store.mysql.MySQLOnlineStore",
-    "hazelcast": "feast.infra.online_stores.contrib.hazelcast_online_store.hazelcast_online_store.HazelcastOnlineStore",
-    "ikv": "feast.infra.online_stores.contrib.ikv_online_store.ikv.IKVOnlineStore",
-    "elasticsearch": "feast.infra.online_stores.contrib.elasticsearch.ElasticSearchOnlineStore",
+    "postgres": "feast.infra.online_stores.postgres_online_store.PostgreSQLOnlineStore",
+    "hbase": "feast.infra.online_stores.hbase_online_store.hbase.HbaseOnlineStore",
+    "cassandra": "feast.infra.online_stores.cassandra_online_store.cassandra_online_store.CassandraOnlineStore",
+    "mysql": "feast.infra.online_stores.mysql_online_store.mysql.MySQLOnlineStore",
+    "hazelcast": "feast.infra.online_stores.hazelcast_online_store.hazelcast_online_store.HazelcastOnlineStore",
+    "ikv": "feast.infra.online_stores.ikv_online_store.ikv.IKVOnlineStore",
+    "elasticsearch": "feast.infra.online_stores.elasticsearch_online_store.ElasticSearchOnlineStore",
     "remote": "feast.infra.online_stores.remote.RemoteOnlineStore",
-    "singlestore": "feast.infra.online_stores.contrib.singlestore_online_store.singlestore.SingleStoreOnlineStore",
-    "qdrant": "feast.infra.online_stores.contrib.qdrant.QdrantOnlineStore",
+    "singlestore": "feast.infra.online_stores.singlestore_online_store.singlestore.SingleStoreOnlineStore",
+    "qdrant": "feast.infra.online_stores.cqdrant.QdrantOnlineStore",
+    **LEGACY_ONLINE_STORE_CLASS_FOR_TYPE,
```

Comment:
IIUC, `LEGACY_ONLINE_STORE_CLASS_FOR_TYPE` would never be used unless someone declares a store like:
```yaml
online_store:
  type: feast.infra.online_stores.contrib.qdrant.QdrantOnlineStore
  ...
```
Is this what you had in mind?

Reply:
agreed, I doubt anyone actually did anything like that. This is technically breaking, but I'm fine with it even w/o leaving aliases.

Reply:
Right that's what I'm referring to. I remember there was somewhere in doc said you can config online_store with something like: `ype: feast.infra.online_stores.contrib.qdrant.QdrantOnlineStore`, but not sure if that's still the case.

Reply:
Anyway let me add the warning to ease the issue.